tensorflow/lite/kernels/arg_min_max.cc

```diff
@@ -37,7 +37,12 @@ constexpr int kOutputTensor = 0;
 
 TfLiteStatus ResizeOutput(TfLiteContext* context, const TfLiteTensor* input,
                           const TfLiteTensor* axis, TfLiteTensor* output) {
-  int axis_value = *GetTensorData<int>(axis);
+  int axis_value;
+  if (axis->type == kTfLiteInt64) {
```

Comment:
Could you add a few words on why this is needed? Something like the one in your PR description is fine.

Reply:
Sure, I will add some notes here.